src/Core/Style.js

```diff
@@ -20,30 +20,31 @@ function baseAltitudeDefault(properties, ctx) {
 }
 
 export function readExpression(property, ctx) {
-    if (property != undefined) {
```

Comment:
Is it ok to not check the validity of property anymore ?

Reply:
Everytime we call this function we always have a check before:
```
if (userValue != undefined) { return readExpression(userValue, style.context); }
```
or
```
if (this.labelDomelement) {
                    content = readExpression(this.labelDomelement, context);
                }
```
Thus i don't see the use to redo the same test afterward.